Tests/UnitTests/Sources/GREYActionsTest.m

```diff
@@ -198,5 +198,36 @@ - (void)testInvalidTapActionSucceedsAfterDisablingConstraints {
   }
 }
 
+- (void)testReplaceTextSucceedsOnUITextField {
+    NSString *textToReplace = @"A String";
+    NSError *error;
+
+    UITextField *textfield = [[UITextField alloc] init];
+
+    id<GREYAction> replace = [GREYActions actionForReplaceText:textToReplace];
+    [replace perform:textfield error:&error];
+
+    XCTAssertNil(error);
+    error = nil;
+
+    XCTAssertEqualObjects(textToReplace, textfield.text);
+}
+
+- (void)testReplaceTextFailsOnUILabel {
+    NSString *textToReplace = @"A String";
+    NSError *error;
+
+    UILabel *label = [[UILabel alloc] init];
+
+    id<GREYAction> replace = [GREYActions actionForReplaceText:textToReplace];
+    [replace perform:label error:&error];
+
+    NSString *reasonSubstring = @"Replace text invoked on an element not of type UITextField.";
```

Comment:
you'd need to make changes to this test when you add uitextfield type as a constraint of the replaceText action.